src/types/filtering/filter.ts

```diff
@@ -37,12 +44,17 @@ const DataFilterNullOrUndefined = t.intersection([
       t.literal("isNotUndefined"),
     ]),
     staticValue: t.union([t.undefined, t.never]),
-    compareField: t.union([t.undefined, t.never]),
   }),
 ]);
 
+const DataFilterFieldCommon = t.type({
+  type: t.literal("dynamicFilter"),
```

Comment:
```suggestion
  filterType: t.literal("DYNAMIC"),
```

Reply:
changed in [94dfe00](https://github.com/pagopa/data-indexer-commons/pull/13/commits/94dfe00f13b3627cfac84c4908a701845eea702b)